WORKSPACE

```diff
@@ -0,0 +1,9 @@
+workspace(name = "com_github_m3db_tally_cpp")
+
+load(":repositories.bzl", "tally_cpp_repositories")
+
+tally_cpp_repositories()
+
+load("@com_github_nelhage_rules_boost//:boost/boost.bzl", "boost_deps")
```

Comment:
anyway we can avoid the dep on boost?

Reply:
Nope, thrift depends on it.

Reply:
whoa, that's such a huge dep to take for a rpc library.

Reply:
Jaeger did this and seriously regretting it (see https://github.com/jaegertracing/jaeger-client-cpp). Consider something else if at all possible.

Reply:
Since Thrift depends on Boost I don't see any way to avoid it.

Reply:
I'd really consider writing the serialization code yourself. Thrift is just that bad.

Reply:
I couldn't agree with you more, it's brought me nothing but pain and misery. With that being said, my hands are tied as it is already the protocol used to serialize M3 metrics

Reply:
Last suggestion: consider maintaining an internal repo with Thrift code to avoid making the open source world deal with it. You can use some other format here if you are willing to add a backend hook for that format (trying to do the same for Jaeger now with Protobuf).

Reply:
I'm not sure I understand, wouldn't we have too rewrite Thrift to avoid using Boost in that case?

Reply:
It's not worth blocking the PR for this. Feel free to open an issue if you want to circle back later.